WordPress/UITestsFoundation/Screens/DomainsScreen.swift

```diff
@@ -2,17 +2,14 @@ import ScreenObject
 import XCTest
 
 public class DomainsScreen: ScreenObject {
-    public let tabBar: TabNavComponent
 
-    let siteDomainsNavbarHeaderGetter: (XCUIApplication) -> XCUIElement = {
+    private let siteDomainsNavbarHeaderGetter: (XCUIApplication) -> XCUIElement = {
         $0.staticTexts["Site Domains"]
     }
 
     var siteDomainsNavbarHeader: XCUIElement { siteDomainsNavbarHeaderGetter(app) }
 
-    public init(app: XCUIApplication = XCUIApplication()) throws {
-        tabBar = try TabNavComponent()
-
+    init(app: XCUIApplication = XCUIApplication()) throws {
```

Comment:
if this screen isn't explicitly called from tests, it doesn't have to be `public`